File: titleCase.js
function titleCase(str) {
    const ans = [];
   let strArr = str.toLowerCase().split(" ");
   for (let i = 0; i < strArr.length; i++){
       strArr[i] = strArr[i].charAt(0).toUpperCase() + strArr[i].slice(1);
       if (strArr[i] == strArr[strArr.length -1]){
            ans.push(strArr[i])
       } else {
           ans.push(strArr[i] + " ");
       }
   }
   console.log(ans.join(""));
}
  
titleCase("I'm a little tea pot");
titleCase("HERE IS MY HANDLE HERE IS MY SPOUT");
titleCase("sHoRt AnD sToUt");

File: title_case.rb
def title_case(str)
  output = ''
  str.split.each do |word|
    word.capitalize!
    output << "#{word} "
  end
  output[-1] = ''
  output
end

p title_case("I'm a little tea pot")
p title_case("HERE IS MY HANDLE HERE IS MY SPOUT")
